concepts.wip/boolean-expressions/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for boolean-expressions concept",
-  "authors": [],
-  "contributors": []
+  "blurb": "A boolean expression is a logical statement that unambiguously evaluates to either true or false",
+  "authors": ["SJ-Nosrat"],
+  "contributors": ["SaschaMann", "cmcaine"]
```

Comment:
Generally contributors are people who have either edited the files afterwards to some extent or who have done a thorough review that significantly impacted the proposed change. Neither is the case here, so you might want to leave it out :)

Reply:
Should I leave the "contributors" array empty?

Reply:
Yes, that seems more fitting here